src/base/orgs/List.svelte

```diff
@@ -13,14 +16,26 @@
     margin: 3rem;
     display: inline-block;
   }
+  .list {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap; 
+  }
 </style>
 
+<div class="list">
 {#each orgs as org}
-  <div class="org">
-    <Link to={`/orgs/${org.address}`}>
-      <Blockies glowOnHover address={org.address} />
-    </Link>
-  </div>
+  {#await Profile.get(org.address, config)}
+    <!-- Avoid using <Loading /> here since it fills the page with loading elements -->
+    <div /> 
+  {:then profile}
+    <div class="org">
+      <Link to={`/orgs/${org.address}`}>
+        <Avatar glowOnHover source={profile.avatar() ?? org.address} />
+      </Link>
+    </div>
+  {/await}
```

Comment:
I think we will eventually want a more efficient way of doing this, as right now it does a bunch of requests for each org :thinking:

Reply:
I think this can be resolved with `Profile.getMulti([string], config)`